home.html

```diff
@@ -0,0 +1,327 @@
+            <p>12 Days Trip</p>
+          </div>
+        </div>
+        <div class="top-selling-card">
+          <div class="card-img">
+            <img src="elements/europe.png" alt="Rome,Italy Image" />
+          </div>
+          <div class="destination-details">
+            <p>Full europe</p>
+            <p>$15k</p>
+          </div>
+          <div class="trip-days">
+            <img src="elements\send-icon.png" />
+            <p>28 Days Trip</p>
+          </div>
+        </div>
+      </main>
+    </section>
+    <section class="book-trip">
+      <section class="left">
+        <header><p>Easy and Fast</p></header>
+        <section class="slogan">
+          <p>Book Your Next Trip <br />In 3 Easy Steps</p>
+        </section>
+        <section class="options">
+          <section class="option">
+            <img
+              src="elements/rectangle.png"
+              alt="rectangle"
+              style="width: 7%"
+            />
+            <div class="trip-content">
+              <h4>Choose Destination</h4>
+              <p class="para">
+                Lorem ipsum dolor sit amet, consectetur adipisicing
+                elit.Urna,tortor tempus.
+              </p>
+            </div>
+          </section>
+          <section class="option">
+            <img src="elements/Group 12.png" alt="payment" />
+            <div class="trip-content">
+              <h4>Make Payment</h4>
+              <p class="para">
+                Lorem ipsum dolor sit amet, consectetur adipisicing
+                elit.Urna,tortor tempus.
+              </p>
+            </div>
+          </section>
+          <section class="option">
+            <img src="elements/Group 11.png" alt="cab" />
+            <div class="trip-content">
+              <h4>Reach Airport on Selected Date</h4>
+              <p class="para">
+                Lorem ipsum dolor sit amet, consectetur adipisicing
+                elit.Urna,tortor tempus.
+              </p>
+            </div>
+          </section>
+        </section>
+      </section>
+      <section class="right">
+        <div class="top-selling-card">
+          <div class="card-img">
+            <img src="elements/girl.jpg" alt="Rome,Italy Image" />
+          </div>
+          <div class="card-texts">
+            <h3>Trip To Greece</h3>
+            <div class="destination-details">
+              <p>14-29 June| by Robbin jo</p>
+            </div>
+            <div class="icons">
+              <img src="elements/LEAF.png" alt="send-icon" />
+              <img src="elements/map.png" alt="" style="width: 25%" />
+              <img src="elements/send.png" alt="send-icon" />
+            </div>
+            <div class="trip-days">
+              <img src="elements\building.png" />
+              <p>24 people going</p>
+              <img
+                src="elements/heart (6) 1.png"
+                alt="like"
+                style="margin-left: 34%; width: 5%"

```

Comment:
in this you have used lot of inline styles please decrease as many as

Reply:
all resolved only 2 left.

Reply:
all resolved